apps/tailwind-components/pages/input/Boolean.story.vue

```diff
@@ -0,0 +1,28 @@
+<template>
+  <InputBoolean id="boolean-story-id" v-model="modelValue1" />
+
+  <div class="my-4">intit empty: {{ modelValue1 }}</div>
+
+  <InputBoolean id="boolean-story-id2" v-model="modelValue2" />
```

Comment:
Removing the enters between the input and results would make this a bit nicer to look at.